src/main/java/io/jenkins/plugins/orka/OrkaComputerLauncher.java

```diff
@@ -132,12 +132,13 @@ private boolean createConfiguration(OrkaAgent agent, OrkaClientProxy clientProxy
             String image = configName;
             String baseImage = agent.getBaseImage();
             int numCPUs = agent.getNumCPUs();
+            String scheduler = agent.getScheduler();
 
             ConfigurationResponse configResponse = clientProxy.createConfiguration(configName, image, baseImage,
-                    template, numCPUs);
+                    template, numCPUs, scheduler);
             if (!configResponse.isSuccessful()) {
                 logger.println(String.format(configurationErrorFormat, Utils.getTimestamp(), configName, image,
-                        baseImage, template, numCPUs, Utils.getErrorMessage(configResponse)));
+                        baseImage, template, numCPUs, scheduler, Utils.getErrorMessage(configResponse)));
```

Comment:
This will fail. The `configurationErrorFormat` field needs to be changed so it can display the scheduler.